jskim5923/ArchitectureStudy/app/src/main/java/com/jskim5923/architecturestudy/main/MainContract.kt

```diff
@@ -0,0 +1,14 @@
+package com.jskim5923.architecturestudy.main
+
+interface MainContract {
+
+    interface View {
+        fun updateViewpager(marketList: List<String>)
+    }
+
+    interface Presenter {
+        fun loadMarketList()
+
+        fun disposableClear()
```

Comment:
함수명은 동사가 먼저 오는 것이 좋습니다.
```suggestion
        fun clearDisposable()
```

Reply:
[9713e85] 적용하였습니다.